SECURITY/SECURITY/main.asm

```diff
@@ -0,0 +1,61 @@
+;
+; SECURITY.asm
+;
+; Created: 10/29/2018 9:14:32 AM
+; Author : PC
+;
+; TRIGGER = 0 (PORTJ) , ECHO = 0 (PORT K), LIGHT = ANY OF THE PORT B () 
+ldi r25, 0xFF; preloading IO mode to output
+ldi r26, 0x00; setting IO mode to input
+ldi r27, 0b00000001; Check this to see if sensor was triggered
+;OUTPUT PORT
+sts ddrk, r26;
+;INPUT PORT
+sts ddrj, r25;
+out ddrb, r25;
+TRIGGERING:
+; Sending low value to the trigger pin to ensure clean HIGH
+sts portj,r26
+;5 US delay
+    ldi  r18, 26
+L1: dec  r18
+    brne L1
+    rjmp PC+1
+
+; Sending high value to the trigger pin
+sts portj,r25						
+;~36ms delay
```

Comment:
I like to see that you have included the execution time of the delay.
Next step could have been to label the delay like `delay_36_ms` and then call it. ..or even better use the calling convention to make the delay length dynamic, so you could avoid having another delay loop later in you program.